src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -99,4 +99,5 @@ endif(WITH_MGR_DASHBOARD_FRONTEND AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch6
 if(WITH_TESTS)
   include(AddCephTest)
   add_tox_test(mgr-dashboard TOX_ENVS lint check)
+  add_test_label(run-tox-mgr-dashboard "long")
```

Comment:
Nothing objectionable here (>200 secs is too much for unit testing), but this leads me to think whether we need more granularity here (I mean breaking down: unit-tests/linting/etc). It could come with some overhead (in the case of run-tox, virtualenv creation), but would allow to run for example minimal/safety checks, instead of full suites.

Reply:
@epuertat
I'm running my jenkins machines in low performance VMs so some of the tests just run longer.

Starting LABELS is exactly the reason for doing this.

Reply:
@tchaikov
I was going for the fact the one can schedule the order of tests manually with the option: 
```
  -I [Start,End,Stride,test#,test#|Test file], --tests-information
                               = Run a specific number of tests by number.
```
And the create a script to do the sorting in `Test file`, depending on the labels.

But I think in essence that `COST` would do what I would like. That would require all tests to get a COST assigned, which is also a clutter in the Cmake files.

Reply:
per https://stackoverflow.com/questions/48547991/what-is-the-default-cost-of-a-ctest-test, ctest uses 0 for the default cost. so if we assign cost to certain tests, the tests without COST will be scheduled to run after them. is this behavior expected?

could you help me understand how you would want to use `--tests-information` for schedule the tests? i feel that it'd be difficult to come up with a sustainable way to manage the "Test File".